client/src/pages/pot-instance/components/AddIngredients/AddIngredients.tsx

```diff
@@ -31,4 +31,4 @@ const AddIngredients = ({ addFoodTimer }: AddFoodTimer) => {
     </>
   );
 };
-export default AddIngredients;
+export default memo(AddIngredients);
```

Comment:
Is this a hook? How is it different from useMemo?

Reply:
`React.memo` is typically used for components, while `useMemo` is for functions. Although you can use `useMemo` for functional components.
`memo` also allows for a function as the second parameter to dictate when a component should re-render, while `useMemo` only has a list of dependencies like `useEffect`.
In this case either method would have been fine, but I wanted to enforce that fact that this component shouldn't need to be re-rendered, when it is a child of any parent.